sdk/core/Azure.Core/tests/ManagementRecordedTestBaseTests.cs

```diff
@@ -203,5 +203,27 @@ public void ValidateForwardClientCallDefault()
             var ex = Assert.ThrowsAsync<InvalidOperationException>(async () => testResource = await testResource.GetForwardsCallDefaultAsync());
             Assert.AreEqual(ex.Message, "Expected some diagnostic scopes to be created, found none");
         }
+
+        [TestCase(RecordedTestMode.Record)]
+        [TestCase(RecordedTestMode.Playback)]
+        [TestCase(RecordedTestMode.Live)]
+        public async Task ValidateWaitOverride(RecordedTestMode mode)
+        {
+            Mode = mode;
+            ManagementTestClient testClient = InstrumentClient(new ManagementTestClient());
+            TestResource testResource = testClient.GetTestResource();
+            Stopwatch sw = Stopwatch.StartNew();
+            testResource = (await testResource.GetLroAsync(WaitUntil.Completed)).Value;
+            sw.Stop();
+            Assert.AreEqual("TestResourceProxy", testResource.GetType().Name);
+            if (mode == RecordedTestMode.Playback)
+            {
+                Assert.LessOrEqual(sw.ElapsedMilliseconds, 1000);
```

Comment:
I think the Assert is ambiguous if both are xOrEqual to 1000

```suggestion
                Assert.Less(sw.ElapsedMilliseconds, 1000);
```